src/main/domain/service/EventStatusService.js

```diff
@@ -43,15 +43,17 @@ export class EventStatusService {
     })
   }
 
-  updateTCLoaded() {
```

Comment:
Do we need to have the default false in more than one place? In LoadUserConsentUseCase and here?

Reply:
I agree with you. Default value only in one place. Thanks and done.